lib/cog/repository/bundles.ex

```diff
@@ -50,6 +50,17 @@ defmodule Cog.Repository.Bundles do
     end
   end
 
+
+  # Deletes the embedded bundle when Mix.env == :dev so Cog can recreate the bundle
+  # at startup.
+  defp __reset_embedded() do
+    if Mix.env == :dev do
```

Comment:
We probably don't need this conditional, since it's also embedded at this function's call site